cypress/integration/racingcar.spec.js

```diff
@@ -72,6 +80,25 @@ describe('자동차 경주 게임 테스트 케이스', () => {
           cy.get('#racing-result h2').contains(winners.join(', '));
         });
     });
+
+    it('경기가 종료된 후, 경기결과를 보여주고 2초 뒤에 축하 알림창을 띄운다.', () => {
+      const alertStub = cy.stub();
+      const racingCount = 5;
+      cy.on('window:alert', alertStub);
+      cy.clock();
+
+      cy.get('#racing-name input').type(carNamesT1);
+      cy.get('#racing-name button').click();
```

Comment:
```suggestion
      cy.get('#racing-name input').type(carNamesT1 + '{enter});
```
type에 추가로 특정 행동을할 수 있게 정해둔 [인자](https://docs.cypress.io/api/commands/type#Arguments)들이 있어요!
submit으로 구성했을 때 enter를 사용하면 별도로 button을 찾아서 클릭하지 않아도 돼서 좋습니다 🙂